yarn-project/acir-simulator/src/client/client_execution_context.ts

```diff
@@ -67,11 +71,14 @@ export class ClientTxExecutionContext {
   private async fetchNotes(contractAddress: AztecAddress, storageSlot: ACVMField, limit: number, offset = 0) {
     const { count, notes } = await this.db.getNotes(contractAddress, fromACVMField(storageSlot), limit, offset);
 
-    const dummyNotes = Array.from({ length: Math.max(0, limit - notes.length) }, () => ({
-      preimage: createDummyNote(),
-      siblingPath: new Array(PRIVATE_DATA_TREE_HEIGHT).fill(Fr.ZERO),
-      index: 0n,
-    }));
+    const dummyNotes = Array.from(
+      { length: Math.max(0, limit - notes.length) },
+      () =>
+        ({
+          preimage: createDummyNote(),
+          index: BigInt(-1),
+        } as NoteLoadOracleInputs),
```

Comment:
rather than a cast at the end I think this could be put as the return type of the closure, which is perferred